test/karma.conf.js

```diff
@@ -10,6 +10,8 @@ module.exports = function(config) {
     files: [
       'lib/jquery/dist/jquery.js',
       'lib/bootstrap-datepicker/dist/js/bootstrap-datepicker.js',
+      'lib/moment/moment.js',
+      'lib/eonasdan-bootstrap-datetimepicker/src/js/bootstrap-datetimepicker.js',
```

Comment:
@jeff-phillips-18  - this has a moment and jquery dependency, does that have any bearing on the work you recently did?